.buildkite/pipelines/esql_grammar_sync.yml

```diff
@@ -0,0 +1,6 @@
+steps:
+  - command: .buildkite/scripts/steps/esql_grammar_sync.sh
+    label: ES|QL Grammar Sync
+    timeout_in_minutes: 10
+    agents:
+      queue: kibana-default
```

Comment:
Can we swap this with n2-2-spot or n2-4-spot? 2 cores 8gb, 4 cores 16gb.  The kibana-default queue leaves agents running for fast startup on short tasks, but generally for anything leaving state on the machine we use fresh instances.

Reply:
Done in [ee55d6c](https://github.com/elastic/kibana/pull/178347/commits/ee55d6cb107c2049d2cd84e836b06b90c4800dea)